Amplify/Core/Model/BasicUserProfile.swift

```diff
@@ -30,7 +30,8 @@ public struct BasicUserProfile: UserProfile {
                 email: String? = nil,
                 plan: String? = nil,
                 location: UserProfileLocation? = nil,
-                customProperties: [String: UserProfilePropertyValue]? = nil) {
+                customProperties: [String: UserProfilePropertyValue]? = nil)
+    {
```

Comment:
There's a lot of these changes that cause new failures because the `{` should go in the previous line. 
I personally prefer to have it in the previous line, this one is giving Objective-C syntax :P

Reply:
Updated the argument wrapping rules, braces formatting should be updated now.